Estore.UITests/Pages/LoginPage.cs

```diff
@@ -0,0 +1,78 @@
+﻿using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+using SeleniumExtras.WaitHelpers;
+using UITests.Pages;
+
+namespace Estore.UITests.Pages
+{
+    public class LoginPage : BasePage
+    {
+        [FindsBy(How = How.CssSelector, Using = "input[type = 'text']")]
+        private IWebElement _emailInputField;
+
+        [FindsBy(How = How.CssSelector, Using = "input[type = 'password']")]
+        private IWebElement _passwordInputField;
+
+        [FindsBy(How = How.CssSelector, Using = "[type='button']")]
+        private IWebElement _loginButton;
+
+        [FindsBy(How = How.XPath, Using = "//input[@type='text']/parent::div/parent::div/following-sibling::div/p")]
+        private IWebElement _errorEmailMessage;
+
+        [FindsBy(How = How.XPath, Using = "//input[@type='password']/parent::div/parent::div/following-sibling::div/p")]
+        private IWebElement _errorPasswordMessage;
+
+        public LoginPage(IWebDriver driver) : base(driver)
+        {
+            Title = "Estore - Login";
+            Wait.Until(d => d.Title == Title);
+        }
+
+        public void FillEmailField(string email)
+        {
+            _emailInputField.Click();
+            _emailInputField.SendKeys(email);
+            _emailInputField.SendKeys(Keys.Tab);
+        }
+
+        public void FillPasswordField(string password)
+        {
+            _passwordInputField.Click();
+            _passwordInputField.SendKeys(password);
+            _emailInputField.SendKeys(Keys.Tab);
```

Comment:
same as above. Here is like workaround of potential bug

Reply:
fixed